### ISSUE TITLE
FC0002: Add support for List/Dictionary

### ISSUE THREAD

**Issue by Arthurvdv** [ALCops/Analyzers#255]
Add support for detecting casing on List and/or Dictionary of types.

```AL
codeunit 50100 MyCodeunit
{
    var
        MyList: List of [INTERFACE "IMYINTERFACE"];
        MyDict: Dictionary of [TEXT, INTERFACE "IMYINTERFACE"];
}

interface "IMyInterface" { }
```

The keywords and declarations in uppercase doesn't raise a diagnostic.
